Description: Add tags to make armhf binaries come out correctly.
 Add elf tag to mark hardfp binaries as such. (Closes: 695547)
Author: Peter Michael Green <plugwash@debian.org>
Bug-Debian: https://bugs.debian.org/695547

Index: fpc/fpcsrc/compiler/arm/agarmgas.pas
===================================================================
--- fpc.orig/fpcsrc/compiler/arm/agarmgas.pas
+++ fpc/fpcsrc/compiler/arm/agarmgas.pas
@@ -132,6 +132,8 @@ unit agarmgas;
         inherited WriteExtraHeader;
         if TArmInstrWriter(InstrWriter).unified_syntax then
           writer.AsmWriteLn(#9'.syntax unified');
+        if target_info.abi = abi_eabihf then
+          writer.AsmWriteLn(#9'.eabi_attribute 28, 1 @Tag_ABI_VFP_args');
       end;
 
 {****************************************************************************}
